src/NUnitEngine/nunit.engine/Internal/ServerUtilities.cs

```diff
@@ -88,12 +95,13 @@ public static TcpChannel GetTcpChannel()
         /// </summary>
         /// <param name="name">The name of the channel</param>
         /// <param name="port">The port to use if the channel must be created</param>
+        /// <param name="currentMessageCounter">An optional counter to provide the ability to wait for all current messages</param>
         /// <returns>The specified <see cref="TcpChannel"/> or null if it cannot be found and created</returns>
-        public static TcpChannel GetTcpChannel( string name, int port )
```

Comment:
The entire file is this way so I followed the local style. I'd like nothing more than to clean up the coding style in the file though. I'll do it in an isolated commit so it isn't so noisy. (@nunit/core-team is this the right approach?)